### PR TITLE
Update downloadIstioCandidate.sh when finalizing release

### PR DESCRIPTION
**Pull request by chxchx**
**Release note**:

```release-note
none
```

### CODE REVIEW

File: toolbox/githubctl/main.go [hunk 1]
```diff
@@ -24,29 +24,28 @@ import (
...
+	nextRelease = flag.String("next_release", "", "Tag of the next release")
+	githubClnt  *u.GithubClient
 )
 
 const (
 	istioVersionFile     = "istio.VERSION"
 	istioDepsFile        = "istio.deps"
 	releaseTagFile       = "istio.RELEASE"
-	downloadScript       = "downloadIstio.sh"
+	downloadScript       = "downloadIstioCandidate.sh"
```
Comment (ldemailly) [inline toolbox/githubctl/main.go]: don't you need it to be `release/downloadIstioCandidate.sh` ?
(I put the new script under `release/` to limit the chance of accidental changes to it - we can't move the other one as the URL is burned)

Comment (chxchx) [reply toolbox/githubctl/main.go]: You are right. I didn't notice the two download scripts are in different directories.

**Review by ldemailly — APPROVED** [review]
I guess it'll get tested for 0.2.5 shortly ?
otherwise how do you test it ?

**Comment by chxchx** [comment]
I tested it with my forked repos. An example PR is here https://github.com/chxchx/istio/pull/35/files.

**Comment by chxchx** [comment]
I tested it with my forked repos. An example PR created by the script is here https://github.com/chxchx/istio/pull/36/files

**Comment by sebastienvas** [comment]
@ldemailly Instead of updating that script for each release, can we just update script to take the latest - like here:
https://github.com/istio/istio/pull/409/files

**Comment by ldemailly** [comment]
@sebastienvas that would slow down the download by adding an extra step for every user running it, if we already modify files when finalizing I think modifying that one is good too

**Pull request merged**